packages/adblocker/src/engine/domains.ts

```diff
@@ -11,6 +11,24 @@ import { toASCII } from '../punycode.js';
 import { StaticDataView, sizeOfUint32Array, sizeOfUTF8 } from '../data-view.js';
 import { binLookup, hasUnicode, HASH_INTERNAL_MULT } from '../utils.js';
 
+export function testNetworkParts(parts: string[]): boolean {
```

Comment:
What is a "network part"? Can we make the naming more specific here as I find it hard to understand without more context.

Reply:
I added an option called `isNetworkEntities` and integrated everything inside of `Domains.prototype.parse`. The new shape looks like the following. Would like you like to change the shape?

```ts
export class Domains {
  public static parse(
    parts: string[],
    {
      isNetworkEntities = false,
      debug = false,
    }: { isNetworkEntities?: boolean | undefined; debug?: boolean | undefined } = {},
  ): Domains | undefined
```

Reply:
outdated